centreon/www/front_src/src/Header/specs/Header.Poller.ts

```diff
@@ -0,0 +1,444 @@
+import { renderHook, act } from '@testing-library/react';
+import { useAtomValue } from 'jotai';
+
+import { userAtom } from '@centreon/ui-context';
+
+import {
+  labelNoLatencyDetected,
+  labelLatencyDetected,
+  labelAllPollers,
+  labelDatabaseUpdateAndActive,
+  labelDatabaseUpdatesNotActive,
+  labelPollerNotRunning,
+  labelDatabaseNotActive,
+  labelConfigurePollers,
+  labelExportConfiguration,
+  labelExportAndReloadTheConfiguration,
+  labelCancel,
+  labelExportAndReload,
+  labelExportingAndReloadingTheConfiguration,
+  labelConfigurationExportedAndReloaded
+} from '../Poller/translatedLabels';
+import { pollerConfigurationPageNumber } from '../Poller/getPollerPropsAdapter';
+import useNavigation from '../../Navigation/useNavigation';
+
+import { initialize } from './Header.testUtils';
+
+const getElements = (): void => {
+  cy.findByRole('button', { name: 'Pollers', timeout: 5000 }).as(
+    'pollerButton'
+  );
+
+  cy.findByRole('status', { name: 'database' }).as('databaseIndicator');
+  cy.findByRole('status', { name: 'latency' }).as('latencyIndicator');
+};
+
+const submenuShouldBeClosed = (label): void => {
+  cy.findByRole('button', { name: label })
+    .as('button')
+    .should('have.attr', 'aria-expanded', 'false');
+
+  cy.get('@button').within(() => {
+    cy.findByTestId('ExpandLessIcon').should('be.visible');
+  });
+  cy.get(`#${label}-menu`).should('not.be.visible').should('exist');
+};
+
+const openSubMenu = (label: string): void => {
+  cy.findByRole('button', {
+    name: label
+  }).click();
+  submenuShouldBeOpened(label);
+};
+
+const submenuShouldBeOpened = (label: string): void => {
+  cy.findByRole('button', { name: label })
+    .as('button')
+    .should('have.attr', 'aria-expanded', 'true');
+
+  cy.get('@button').within(() => {
+    cy.findByTestId('ExpandMoreIcon').should('be.visible');
+  });
+  cy.get(`#${label}-menu`).should('be.visible').should('exist');
+};
+
+export default (): void =>
+  describe('Pollers', () => {
+    describe('responsive behaviors', () => {
+      it('should hide button text at smaller screen size', () => {
```

Comment:
Please, read this documentation in order to write test names: https://centreon.atlassian.net/wiki/spaces/RD/pages/416612399/General+guidelines#Testing